tests/ntp/test_ntp.py

```diff
@@ -11,44 +11,46 @@
     pytest.mark.device_type('vs')
 ]
 
+
 @pytest.fixture(scope="module")
 def setup_ntp(ptfhost, duthost, creds):
     """setup ntp client and server"""
-    if creds.get('proxy_env'):
-        # If testbed is behaind proxy then force ntpd inside ptf use local time
-        ptfhost.lineinfile(path="/etc/ntp.conf", line="server 127.127.1.0 prefer")
 
-    # enable ntp server
-    ntp_en_res = ptfhost.service(name="ntp", state="started")
+    ptfhost.lineinfile(path="/etc/ntp.conf", line="server 127.127.1.0 prefer")
```

Comment:
are we always forcing the ntpd to use local time now?

Reply:
Yes. This change will force the ntpd on PTF to always use local time. Then the ntp server on PTF docker will always be "synchronized". This test will be more stable. Since we are not updating time of the PTF container, time of PTF will be same as host. Eventually, time of DUT will be kind of synchronized with host.